splink/files/chart_defs/blocking_rule_generated_comparisons.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
+
+    "config": {
+      "view": {
+        "width": 450,
+        "height": 200
+      },
+      "mark": {
+        "tooltip": null
+      },
+      "title": {
+        "anchor": "middle"
+      },
+      "header": {
+        "title": null
+      }
+    },
+
+    "data": {
+      "values": []
+    },
+
+    "mark": "bar",
+    "title": {
+      "text": "Blocking Rule Comparisons Generated by Blocking Rules",
+      "subtitle": ""
```

Comment:
```suggestion
      "subtitle": "(Counts exclude comparisons already generated by previous rules)"
```

Reply:
I think this is more important for clarity than including 'total comparisons generated' in the title (which is accessible  in the tooltip)